src/core/compact_object.h

```diff
@@ -103,7 +105,7 @@ class CompactObj {
     EXPIRE_BIT = 2,
     FLAG_BIT = 4,
 
-    // ascii encoding is not an injective function. it compresses 8 bytes to 7 but also 7 to 7.
+    // ascii encoding is not an indicative function. it compresses 8 bytes to 7 but also 7 to 7.
```

Comment:
what's this ?

Reply:
should not be there, will remove - not sure what this is (maybe I did spell check but this is not my comment).